### PR TITLE
Update SS58 Codec documentation

### PR DESCRIPTION
**Pull request by gabrielokura**
## Description
This PR updates package ss58_codec README and example file.

## Why?
Completing first milestone. 

## How?
1. Update example.dart
2. Update README.

## Testing Plan 
Run `dart test` in ss58_codec package folder

### CODE REVIEW

**Comment by leonardocustodio** [comment]
Would be nice to show the option to encode with a prefix that the user wants

**Review by leonardocustodio — CHANGES_REQUESTED** [review]
above

File: packages/ss58_codec/README.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+
+void main() {
+  // Decoding a Polkadot Address Object
+  Address address =
+      SS58Codec.decode('1zugcag7cJVBtVRnFxv5Qftn7xKAnR6YJ9x4x3XLgGgmNnS');
+  print('Address: $address');
+
+  // Encoding an Address Object
+  String encodedAddress = SS58Codec.encode(address);
+  print('Encoded address: $encodedAddress');
```
Comment (Lohann) [inline packages/ss58_codec/README.md:22]: Where the SS58 prefix is defined?

Comment (gabrielokura) [reply packages/ss58_codec/README.md]: Add one more example to encode custom addresses, but this one is encoding previous decoded address

File: packages/ss58_codec/README.md [hunk 2]
```diff
@@ -1,3 +1,31 @@
-## ss58_codec
+## SS58 Codec
 
-In Progress...
+Provides encoder and decoder used by [SS58](https://github.com/rankanizer/polkadart/tree/main/packages/ss58) to parse [substrate](https://docs.substrate.io/fundamentals/accounts-addresses-keys/) addresses.
+
+## Usage
+
+To use this package, add `ss58_codec` as a [dependency in your pubspec.yaml file](https://docs.flutter.dev/development/platform-integration/platform-channels).
```
Comment (pastre) [inline packages/ss58_codec/README.md:7]: Is this published to pub?
Can we publish it without creating it as a repository?

**Review by leonardocustodio — APPROVED** [review]

File: packages/ss58_codec/README.md (hunk 2 above)
Comment (gabrielokura) [reply packages/ss58_codec/README.md:7]: Not yet. We can publish it using polkadart repository as reference

Comment (gabrielokura) [reply packages/ss58_codec/README.md:7]: After the merge, i can publish the package to pub.dev

**Review by leonardocustodio — APPROVED** [review]

File: packages/ss58_codec/pubspec.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 name: ss58_codec
-description: ss58_codec description
+description: Provides encoder and decoder used to parse substrate addresses.
```
Comment (justkawal) [inline packages/ss58_codec/pubspec.yaml:2]: [Suggestion]
description: Encodes and decodes substrate addresses.

**Review by justkawal — APPROVED** [review]
Just a suggestion,
Others LGTM

**Pull request merged**